components/Steps/CTAStep.tsx

```
@@ -1,16 +1,24 @@
 import { Tabs, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import { type CTAStep } from "@/types";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "../ui/select";
 
-const CTAStep = () => {
+const CTAStep = ({ stripeProducts }: CTAStep) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'CTAStep'. Consider to delete it or rename it.

'CTAStep' is defined here:


(lint/suspicious/noRedeclare)